src/Appwrite/Migration/Migration.php

```diff
@@ -93,6 +93,7 @@ abstract class Migration
         '1.6.0' => 'V21',
         '1.6.1' => 'V21',
         '1.6.2' => 'V22',
+        '1.7.0' => 'V23',
```

Comment:
We can remove for now. We can do this later when all is merged in 1.7.x

Reply:
Tests are breaking without it.